x-pack/plugins/apm/public/components/shared/service_link.tsx

```diff
@@ -35,22 +52,61 @@ export function ServiceLink({
     ? '/mobile-services/{serviceName}/overview'
     : '/services/{serviceName}/overview';
 
-  return (
-    <StyledLink
-      data-test-subj={`serviceLink_${agentName}`}
-      href={link(serviceLink, {
-        path: { serviceName },
-        query,
-      })}
-    >
-      <EuiFlexGroup alignItems="center" gutterSize="s" responsive={false}>
+  if (serviceName === OTHER_SERVICE_NAME) {
+    return (
+      <EuiFlexGroup alignItems="center" gutterSize="xs">
         <EuiFlexItem grow={false}>
-          <AgentIcon agentName={agentName} />
+          <EuiText
+            grow={false}
+            style={{ fontStyle: 'italic', fontSize: '1rem' }}
+          >
+            _other
+          </EuiText>
         </EuiFlexItem>
-        <EuiFlexItem className="eui-textTruncate">
-          <span className="eui-textTruncate">{serviceName}</span>
+        <EuiFlexItem>
+          <PopoverTooltip
+            ariaLabel={i18n.translate('xpack.apm.serviceLink.tooltip', {
+              defaultMessage: 'Max service groups reached tooltip',
```

Comment:
I don't think we should have "tooltip" in a tooltip.

Reply:
Lol, good catch :D